spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe "Movie Show" do
+  before :each do
+    @studio_1 = Studio.create!(name: "MAPPA Studio", location: "Suginami")
+
+    @movie_1 = @studio_1.movies.create!(title: "Jujutsu Kaisen", creation_year: "2018", genre: "Adventure Dark", studio_id: @studio_1.id)
+
+    @actor_1 = Actor.create!(name: "Yuji Itadori", age: 15)
+    @actor_2 = Actor.create!(name: "Gojo Satoru", age: 29)
+    @actor_3 = Actor.create!(name: "Megumi Fushiguro", age: 15)
+    @actor_4 = Actor.create!(name: "Maki Zenin", age: 16)
+  end
+
+  describe "US 2- Movie Show Page" do
+    describe "When i visit a movie page I see the movies info" do
+      it "Also displays all actors in it from youngest to oldest and I see the average age of all the actors" do
+        @movie_1.actors << [@actor_1, @actor_2, @actor_3, @actor_4]
+        visit "/movies/#{@movie_1.id}"
+
+        expect(page).to have_content(@movie_1.title)
+        expect(page).to have_content(@movie_1.creation_year)
+        expect(page).to have_content(@movie_1.genre)
+
+        expect(page).to have_content(@actor_1.name)
```

Comment:
You can add `orderly` to the gem file test group as well and use `appear_before` to test order here